### ISSUE TITLE
Sfsymbols don't respect neither sfsize nor size

### ISSUE THREAD

**Issue by fabiospampinato** [swiftbar/SwiftBar#245]
I have this item:

```js
console.log ( `Asd | sfimage=heart.fill size=8 sfsize=8 shell="${__dirname}/like.sh" terminal=false` );
```

Which is rendered like this (i.e., the icon is always the same size):

<img width="51" alt="Screen Shot 2021-11-13 at 23 59 41" src="https://user-images.githubusercontent.com/1812093/141662495-c7f99c8f-f6cb-4731-9d32-0ac610a01316.png">

**Comment by melonamin** [comment]
`image` and `sfimage` are always the same size - default size for [menu bar item](https://developer.apple.com/documentation/appkit/nsstatusitem)

`size` and `sfsize` affect only embeded SF Symbols in text, like this: `Test :cloud: | sfsize = 30`

**Comment by fabiospampinato** [comment]
I see, is that an intentional behavior? It sounds pretty confusing to me 🤔

**Comment by fabiospampinato** [comment]
Like what which scenario I would want my embedded symbols to have one size and the other symbol to have another size?

**Comment by melonamin** [comment]
> I see, is that an intentional behavior? It sounds pretty confusing to me

Yes it is, it described in the Readme as well. We got here through a burn of releases, I can see how it can be confusing to a newcomer.

> Like under what scenario would I want my embedded symbols to have one size and the other symbol to have another size?

It is more like you have two options:

- Have a standard look, same as other menu bar items: `Image and Text | sfiamge\Image=....`
- Have a customized look, non-standard text size, image not at the start or more than one image: `Text and :image: and another :image: | size=... sfsize=...`

**Comment by fabiospampinato** [comment]
> Have a standard look, same as other menu bar items: Image and Text | sfiamge\Image=....

FWIW the standard look for me meant that sfsymbols were much much bigger than all the standard ones, hence me trying to change the size of them.

**Comment by melonamin** [comment]
Looking at the code, apparently I have `width` and `height` params for `image`, but not `sfimage`. If I add will it address the issue?

```swift
var image: NSImage? {
    if #available(OSX 11.0, *) {
        if let sfString = params["sfimage"] {
            let config = NSImage.SymbolConfiguration(scale: .large)
            return NSImage(systemSymbolName: sfString, accessibilityDescription: nil)?.withSymbolConfiguration(config)
        }
    }

    let image = NSImage.createImage(from: params["image"] ?? params["templateimage"], isTemplate: params["templateimage"] != nil)
    if let widthStr = params["width"], let width = Float(widthStr),
       let heightStr = params["height"], let height = Float(heightStr)
    {
        return image?.resizedCopy(w: CGFloat(width), h: CGFloat(height))
    }

    return image
}
```

**Comment by fabiospampinato** [comment]
Possibly 🤔
